core/src/net/sf/openrocket/rocketcomponent/RailButton.java

```diff
@@ -66,7 +66,7 @@ public RailButton(){
 		this.totalHeight_m = 0.0097;
 		this.innerDiameter_m = 0.008;
 		this.flangeHeight_m = 0.002;
-		this.setStandoff( 0.002);
```

Comment:
This change doesn't seem relevant to the current commit. Don't get me wrong, I prefer the spacing changes you have provided, however the change is irrelevant for this issue and we should address some of the formatting issues in subsequent patches.

Reply:
Although I agree with your point of view, I also think it's easier to refactor some things when you see them, instead of having to do everything at once in a later patch. But that's just an opinion, it does make it harder to see what has really changed in a commit when refactoring is included.